modules/linear_boltzmann_solvers/b_discrete_ordinates_solver/sweep/angle_aggregation/angle_aggregation.cc

```diff
@@ -10,22 +10,23 @@ namespace lbs
 {
 
 AngleAggregation::AngleAggregation(
-  const std::map<uint64_t, std::shared_ptr<SweepBndry>>& in_sim_boundaries,
-  size_t in_number_of_groups,
-  size_t in_number_of_group_subsets,
-  std::shared_ptr<AngularQuadrature>& in_quadrature,
-  std::shared_ptr<MeshContinuum>& in_grid)
+  const std::map<uint64_t, std::shared_ptr<SweepBndry>>& sim_boundaries,
```

Comment:
I know this is not a change made in this PR, but it would be nice to just spell out `SweepBoundary`. Additionally, any idea what a `sim_boundary` is versus a `boundary`? Feel free to punt on this.

Reply:
I address SweepBndry -> SweepBoundary in a future commit.  I'll also addres sim_boundaries there as well.